templates/skeleton/app/routes/collections/index.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  useLoaderData,
+  Link,
+  useRouteError,
+  isRouteErrorResponse,
+  useCatch,
+} from '@remix-run/react';
+import {
+  json,
+  type LoaderArgs,
+  type ErrorBoundaryComponent,
+} from '@shopify/remix-oxygen';
+import {type CollectionConnection} from '@shopify/hydrogen/storefront-api-types';
+import {Image} from '@shopify/hydrogen';
+
+export async function loader({context}: LoaderArgs) {
+  const {collectionConnection} = await context.storefront.query<{
+    collectionConnection: CollectionConnection;
+  }>(COLLECTIONS_QUERY);
+
+  return json({collectionConnection});
+}
+
+export default function Collections() {
+  const {collectionConnection} = useLoaderData<typeof loader>();
+
+  return (
+    <>
+      {collectionConnection.nodes.map((collection, collectionIndex) => {
+        return (
+          <div key={collection.id}>
+            <Link to={`/collections/${collection.handle}`}>
+              <h3>{collection.title}</h3>
+            </Link>
+            {collection.image && (
+              <Image
+                data={collection.image}
+                loading={collectionIndex === 0 ? 'eager' : undefined}
```

Comment:
`undefined` or `'lazy'` as the default? They achieve the same thing, but I was wondering if using `undefined` would help teach that `'lazy'` is the default?

Reply:
Good point; would you prefer to leave it as undefined but have a comment?